app/views/juntos_bootstrap/projects/_project_header.html.slim

```diff
@@ -24,7 +24,7 @@
         = @project.headline
     - unless @project.subgoals.empty?
       = render 'juntos_bootstrap/projects/subgoals'
-    - if %w(juntosGSA2015 aprendaladaca gsa2015 corretordobem movimentoperfeito bibliotecadamell fundojovem pesquisadoacao tempodebrincar2015 afabricadedoces adoteumaluno cidadedosmeninos asa festadenatal associacaoacorde bolsasdazmina marcha2016 paranaue jornaljoca projetooca amigosdobem).include?(@project.permalink.downcase)
+    - if %w(juntosGSA2015 aprendaladaca gsa2015 corretordobem movimentoperfeito bibliotecadamell fundojovem pesquisadoacao tempodebrincar2015 afabricadedoces adoteumaluno cidadedosmeninos asa festadenatal associacaoacorde bolsasdazmina marcha2016 paranaue jornaljoca projetooca amigosdobem tuccamcdia).include?(@project.permalink.downcase)
```

Comment:
estou atrasado, mas eu colocaria essa lista dentro do secrets, do .env ou o que for que vocês estejam usando de configuração. Hardcoded assim não é uma boa.